lib/plugins/pluginUtils.js

```diff
@@ -114,7 +114,9 @@ function createProcessAttribute(fn, attributeType) {
             } else if (attribute.metadata) {
                 Object.keys(attribute.metadata).forEach(function(key) {
                     var entry = attribute.metadata[key];
-                    entry.value = fn(entry.value);
+                    if(entry.type === 'DateTime'){
```

Comment:
Entry in CHANGE_NEXT_RELEASE about this fix should be added (mentioning issue #869), pls.

Reply:
Fixed 4d54070